student/src/main/java/tds/student/util/remote/RemoteToLegacyEqualityUtility.java

```diff
@@ -0,0 +1,70 @@
+package tds.student.util.remote;
+
+import java.util.List;
+
+import tds.student.services.data.ItemResponse;
+import tds.student.services.data.PageGroup;
+
+/**
+ * This is a simple equality utility class to compare objects created by legacy
+ * code and those created from remote objects.  The existing domain objects have
+ * a different equals/hashcode implementation which at this point we're afraid
+ * to modify
+ */
+public class RemoteToLegacyEqualityUtility {
```

Comment:
This is really only here to help with tracking down problematic data differences in our code.  The equals methods in the client objects don't ensure every field is the same.  Rather than modify those objects I used this to track down all the data differences I was seeing.

I'd like to leave this in for a while because I think this will be helpful for me when I debug item selection issues.